### ISSUE TITLE
Ability to run through the card list and mass update a subset of information

### ISSUE THREAD

**Issue by kushieda-minori** [kushieda-minori/vc_file_grouper#50]
Breakout of #45 

Ability to run through the card list and mass update a subset of information. Depends on #47, #48, #49

* [x] Allow "manual review" and "full automation". 
* [x] Full automation should generate report at the end of all changes made during the run.
* [x] Manual Review should have an option to switch to auto from the current starting point in the list.
* [ ] The list of cards should be filterable (run off of the table filter?)

**Comment by kushieda-minori** [comment]
game is over. won't fix
